lib/block_io.php

```diff
@@ -83,6 +83,15 @@ private function _request($path, $args = array(), $method = 'POST')
 
         // Initiate cURL and set headers/options
         $ch  = curl_init();
+        
+        // If we run windows, make sure the needed pem file is used
+        if(strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+        	$pemfile = dirname(realpath(__FILE__)) . DIRECTORY_SEPARATOR . 'cacert.pem';
```

Comment:
Yes, __DIR__ was added first in PHP 5.3. dirname(__FILE__) is the equivalent before that.